rhel/zsh/.profile

```diff
@@ -82,3 +82,8 @@ alias code="flatpak run com.visualstudio.code"
 export PKG_CONFIG_PATH="/usr/lib64/pkgconfig:${PKG_CONFIG_PATH}"
 # Ensure path for GO programming language
 export GOPATH="${HOME}/go"
+
+# Check if the .wslprofile file exists, then source it
```

Comment:
Will compile a single .zshrc file. Everything that are OS/Distro-related will be implemented on `.profile`

Reply:
I don't suggest that as it could increase instability & refactor work